src/services/sync/strategies/WebUntis/WebUntisSchoolyearSyncer.js

```diff
@@ -0,0 +1,483 @@
+const assert = require('assert');
+
+const WebUntisBaseSyncer = require('./WebUntisBaseSyncer');
+
+
+/**
+ * Implements syncing from WebUntis API based on the Syncer interface
+ * @class WebUntisSchoolyearSyncer
+ * @implements {Syncer}
+ */
+class WebUntisSchoolyearSyncer extends WebUntisBaseSyncer {
+	/**
+    * Constructor
+    *
+    * Disabled because of ESlint
+    *
+    * @param {*} app
+    * @param {*} stats
+    * @param {*} logger
+	* @param {*} data
+	* 
+	* Current data structure: {
+	*	datasourceId: associated SchuldCloud data source id,
+	*	username: Username for the user to access WebUntis,
+	*	password: Password for the user to access WebUntis,
+	*	url: URL of the WebUntis endpoint,
+	*	datatype: 'inclusive' or 'exclusive', depending on the intended semantics for courseMetadataIds
+	*	courseMetadataIds: list of metadata IDs to consider for inclusion/rejection
+	*	dryrun: collect metadata instead of synching
+	* }
+    */
+	/* constructor(app, stats, logger, data) {
+		super(app, stats, logger, data);
+	} */
+
+	/**
+	 * @see {Syncer#respondsTo}
+	 */
+	static respondsTo(target) {
+		return target === 'webuntis-schoolyear';
+	}
+
+	static params(params, data = {}) {
+		// TODO: check usage of params. params.query, and data
+		
+		const query = (params || {}).query || {};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['query' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4061937)